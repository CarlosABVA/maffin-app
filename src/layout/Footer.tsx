import React from 'react';

export default function Footer(): JSX.Element {
  const currentYear = new Date().getFullYear();

  return (
    <footer className="bottom-0 w-full fixed flex justify-end bg-gunmetal-700 text-sm px-3 z-1">
      {currentYear}
      Â©
      {' '}
      maffin.io
    </footer>
  );
}
