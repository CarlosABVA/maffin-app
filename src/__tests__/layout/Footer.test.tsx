import React from 'react';
import { render } from '@testing-library/react';

import Footer from '@/layout/Footer';

describe('Footer', () => {
  it('renders as expected', () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
