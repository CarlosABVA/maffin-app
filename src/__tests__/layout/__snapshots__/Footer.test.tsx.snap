// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as expected 1`] = `
<div>
  <footer
    class="bottom-0 w-full fixed flex justify-end bg-gunmetal-700 text-sm px-3 z-1"
  >
    2023
    Â©
     
    maffin.io
  </footer>
</div>
`;
